components/ActionSheet/docs/theming.md

```diff
@@ -0,0 +1,54 @@
+### Theming
+
+You can theme an MDCActionSheet to match the Material Design style by using a theming extension. The content below assumes you have read the article on [Theming](../../docs/theming.md).
```

Comment:
I think this needs one more level of unnesting.

Reply:
Done